pynetinstall/flash.py

```diff
@@ -327,30 +327,31 @@ def do_file(self, file: io.BufferedReader, max_pos: int, file_name: str) -> None
             else:
                 # main reason why flashing is so slow but without this sleep state errors occur
                 time.sleep(0.005)
-        
+
     def do_files(self) -> None:
         """
         Sends the npk and the rsc file to the Connection using the do_files() Function
         It requests both files from the get_files() Function of the Plugin
         """
-        npk, rsc = self.plugin.get_files(self.info)
-        if npk is None:
-            raise AbortFlashing("Plugin did not return RouterOS.")
+        files = self.plugin.get_files(self.info)
+        for npk in files[:-1]:
```

Comment:
```suggestion
        *npks, rsc = self.plugin.get_files(self.info)
        if not all(npks):
            raise AbortFlashing("Plugin did not return RouterOS or an additional package.")
        for npk in npks:
```

(and removing line 354)

this has the benefit that if the plugin returns only a single-element tuple, we won't accidentally try to install the system.npk as a config file